lib/more_core_extensions/core_ext/object/descendants.rb

```diff
@@ -0,0 +1,17 @@
+require 'active_support/core_ext/class/subclasses'
+
+module MoreCoreExtensions
+  module Descendants
+    #
+    # Retrieve a descendant by its name
+    #
+    def descendant_get(desc_name)
+      return self if desc_name == to_s || desc_name.nil?
```

Comment:
Should we use `:name` rather than `:to_s` here for consistency?
Why would this be called with `desc_name = nil`?